packages/amplify-category-function/src/provider-utils/awscloudformation/utils/addLayerToFunctionUtils.ts

```diff
@@ -49,9 +50,19 @@ export const askLayerSelection = async (
       askArnQuestion: true,
     };
   }
+
+  const disabledMessage = 'Layer requires migration. Run "amplify update function" and choose this layer to migrate.';
```

Comment:
Can we use `amplify function update` as that's the by-design order, nevertheless it is working the other way around as well...